integration-test/gradle.properties

```diff
@@ -2,7 +2,7 @@
 # Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
 #
 
-mainKotlinVersion=1.4.10
+mainKotlinVersion=1.4.30-M1
```

Comment:
Don't forget to change `mainLibVersion` to `1.1.0-SNAPSHOT` so your optimizations will be enabled